modules/custom/az_person/config/install/core.entity_view_display.node.az_person.default.yml

```diff
@@ -19,11 +19,211 @@ dependencies:
     - field.field.node.az_person.field_az_titles
     - node.type.az_person
   module:
+    - field_group
     - file
     - link
+    - smart_title
     - telephone
     - text
     - user
+third_party_settings:
+  smart_title:
+    enabled: true
+    settings:
+      smart_title__link: false
+      smart_title__tag: h1
+      smart_title__classes:
+        - node__title
+        - d-none
+        - d-md-block
+        - my-0
+        - text-blue
+  field_group:
+    group_mobile_node_title:
+      children:
+        - field_az_fname
+        - field_az_lname
+      label: 'Mobile - Node Title'
+      parent_name: group_col_12_name_titles
+      region: content
+      weight: 6
+      format_type: html_element
+      format_settings:
+        classes: 'node__title d-md-none text-blue mb-0'
+        show_empty_fields: false
+        id: ''
+        element: h1
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_row:
+      children:
+        - group_col_12_name_titles
+        - group_col_12_image
+        - group_col_bio
+      label: Row
+      parent_name: ''
+      region: content
+      weight: 0
+      format_type: html_element
+      format_settings:
+        classes: 'row d-md-block'
```

Comment:
```suggestion
        classes: 'row d-md-block clearfix mb-5'
```

This PR is looking good to me, but I noticed one more small issue in Probo. Currently, the template for the default display has a [containing div with the mb-5 class](https://github.com/az-digital/az_quickstart/blob/c0a9ef942932592337acfc0882f95371558566d6/themes/custom/az_barrio/templates/content/node--az-person--full.html.twig#L112) to add a bottom margin below all the content. That margin seems to be missing on the Probo site, as shown in the After screenshot below. To fix this, we could add the `mb-5` class to the Row div, but we'll need to add the `clearfix` class as well to account for the floated elements.

[Before](https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1979.probo.build/person/wilbur-wildcat):
<img width="476" alt="image" src="https://user-images.githubusercontent.com/74572157/195459462-e6ddcfdc-7455-4a6b-9774-78acb1c7b8d2.png">

[After this PR](https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1989.probo.build/person/wilbur-wildcat): 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/74572157/195459590-5c27770d-8d62-40cd-98b6-92e4b0b4bf80.png">